### DISCUSSION TITLE
What is the current video-meeting schedule?

### DISCUSSION THREAD

**Discussion by ysangkok** [mit-dci/utreexo#239] [Q&A] [answered]
I have an event in my calendar for today, but it seems like it actually happened yesterday.

**Comment by kcalvinalvin** [comment, marked as answer]
Ah yeah we moved it to Monday. It’s on every Monday at 9am est

**Comment by adiabat** [comment]
We can certainly change the day / time to whenever.  I can't do Tuesdays at 9AM US eastern anymore but can do other days.  Right now we have Europe, eastern US and Korea, so no matter what someone's late at night.  The current time doesn't work well for US west coast though.
